ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -19,6 +19,38 @@
 #define MI_POOL_COPY_BYTES    512
 #define MI_MAX_TRANSFER_SIZE  64 * 1024
 
+#if _MI_PAGING_LEVELS == 2
```

Comment:
`#if` could be inside the function.
Do we want to add parentheses?

Reply:
This was a copy of the original deleted code as it existed before the problem commit. It was originally in 'ntoskrnl/mm/ARM3/miarm.h' and  I moved it here to be able to keep all of this code in a single file. This pattern using the ```__MI_PAGAING_LEVELS``` appears in this same pattern in other places in the code, so I feel that it should not be modified here. Thanks.